### ISSUE TITLE
Background and foreground colors have a sufficient contrast ratio error axe-core Error:Cannot read property 'cells' of undefined

### ISSUE THREAD

**Issue by bhaskar1603** [GoogleChrome/lighthouse#12039]
#### Provide the steps to reproduce
1. Run LH on https://gurugramproperty.in/

#### What is the current behavior?
shows red Question mark on accessibility test for both mobile and desktop report
Background and foreground colors have a sufficient contrast ratio error axe-core Error (Cannot read property 'cells' of undefined)

#### What is the expected behavior?
Shall show the accessibility score .

#### Environment Information
* Lighthouse version: 6.4.0
* Device: Emulated Desktop
Network throttling: 40 ms TCP RTT, 10,240 Kbps throughput (Simulated)
CPU throttling: 1x slowdown (Simulated)
Channel: devtools

User agent (host)
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.104 Safari/537.36

User agent (network)
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4143.7 Safari/537.36 Chrome-Lighthouse

CPU/Memory Power
1483

Axe version
3.5.5

**Related issues**

**Comment by patrickhulce** [comment]
Thanks for filing @bhaskar1603! This appears to have already been fixed in Lighthouse 7.0 with the latest version of axe. You can use Chrome Canary for now as a workaround or if you're still able to repro with axe-core, welcome to file an issue [over there](https://github.com/dequelabs/axe-core) :)

**Comment by patrickhulce** [comment]
I can't reproduce this with the settings you specified in #12043 either. Is there any additional information you can provide? Any additional steps or settings used? Is it possible there's a local extension being used that causes this to happen?

Using just axe-core I also cannot reproduce this error.

**Comment by bhaskar1603** [comment]
I didn't used any additional step. can you please specify what else info can i provide
![Screenshot (52)](https://user-images.githubusercontent.com/61049327/106778519-1339fa80-666c-11eb-87ae-9fa9c57a8aae.png)

**Comment by bhaskar1603** [comment]
i don't have any extentions installed

**Comment by patrickhulce** [comment]
I still can't reproduce :/ but there appears to be a nearly identical error open bug at https://github.com/dequelabs/axe-core/issues/2674. If it's the same root issue, when axe-core fixes it Lighthouse will be fixed on the next update.

**Comment by Scrabble96** [comment]
I've just had this on Google Chrome Version 88.0.4324.150 (Official Build) (64-bit) and on Microsoft Edge Version 88.0.705.68 (Official build) (64-bit) on Windows 10 Pro. Both are using Lighthouse 6.4.0.

_This did not happen last week._ 

![Lighthouse-axe-core-error](https://user-images.githubusercontent.com/36987494/107882661-8b14ea00-6ee2-11eb-93ec-512cdc80be75.jpg)

**Comment by phamhuanmko** [comment]
My website also has this error! Who can help me!
![image](https://user-images.githubusercontent.com/45422035/113657593-78d85080-96c8-11eb-813c-70823cfbb292.png)
My web is: https://hungthinhclinic.com/
Thank all

**Comment by gwenevieve** [comment]
Also having this error on one of my sites. 'axe-core cannot read property '4' of undefined'.

**Comment by Dawntraoz** [comment]
In my articles page is also happening https://www.dawntraoz.com/blog/my-web-performance-journey-with-nuxt-storyblok-netlify/.
When I pass the AxeDevtools it shows me 0 issues but in lighthouse I get the question mark.
The problem is the scrollable divs I have for code blocks, when I removed them everything it works just fine.

If I can help with some debugging let me know :)

**Comment by Dawntraoz** [comment]
For me, it already works in the last 8 version!! 🏄🏽‍♀️  Thank you folks!

**Comment by Lucania001** [comment]
Still the same issue on https://zenbears.co.uk. We are 90-100 on everything so this is frustrating

**Comment by bluepeter** [comment]
This is definitely still an issue for us on various sites/pages.

**Comment by Lucania001** [comment]
To confirm I fixed the issue with zenbears. Mine was my offscreen nav was causing havoc. Make sure any off screen elements that you only want rendered with JS or on click have display none set

**Comment by digiage** [comment]
The same error for [https://digiage.cz](https://digiage.cz). Chrome 94, Lighthouse 8.3, Axe 4.2.3
